lighthouse-core/runner.js

```diff
@@ -111,7 +111,14 @@ class Runner {
 
       /** @type {Object<string, LH.Audit.Result>} */
       const resultsById = {};
-      for (const audit of auditResults) resultsById[audit.name] = audit;
+      for (const audit of auditResults) {
+        resultsById[audit.name] = audit;
+
+        if (audit.warnings.length) {
+          const prefixedWarnings = audit.warnings.map(msg => `${audit.description}: ${msg}`);
+          lighthouseRunWarnings.push(...prefixedWarnings);
```

Comment:
all warnings are pushed to the toplevel warnings?

relatedly: i haven't seen a top level warning in the report in a long time. :)

Reply:
> all warnings are pushed to the toplevel warnings?

correct, I figure we can always change this without breaking LHR changes later if we don't like it/prefer to flag the individual audits, buuuuuut as you point out...

> relatedly: i haven't seen a top level warning in the report in a long time. :)

agreed I can't remember seeing one naturally, so seems like a good thing to try this out